src/backend/services/transaction-api/tests/functional/test_routes.py

```diff
@@ -138,6 +141,9 @@ def test_create_transaction_correct_payload(test_client, json_header, requests_m
     requests_mock.post("http://blockchain:5000/wallet/verifyAmount",
                        json={"valid": True}, status_code=200)
 
+    requests_mock.post("http://blockchain:5000/wallet/checkWallet",
```

Comment:
duplicate to line 126?

Reply:
Yeah that endpoint needs to be mocked for both tests